twilight-lavalink/src/model/outgoing.rs

```diff
@@ -0,0 +1,453 @@
+//! Events that clients send to Lavalink.
+use serde::{Deserialize, Serialize};
+use twilight_model::{
+    gateway::payload::incoming::VoiceServerUpdate,
+    id::{marker::GuildMarker, Id},
+};
+
+/// The track on the player. The encoded and identifier are mutually exclusive.
+/// We don't support userData field currently.
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
```

Comment:
IMO we should not derive deserialize here (and neither should be derive serialize for incoming models)

Reply:
I don't understand why that matters?